Client/Frontend/Fakespot/Views/NoAnalysisCardView.swift

```diff
@@ -0,0 +1,153 @@
+        static let bodyLabelFontSize: CGFloat = 13
+        static let footerLabellFontSize: CGFloat = 13
+        static let contentStackViewSpacing: CGFloat = 8
+        static let contentStackViewPadding: CGFloat = 16
+    }
+
+    private var viewModel: NoAnalysisCardViewModel?
+
+    private lazy var cardContainer: ShadowCardView = .build()
+
+    private lazy var contentStackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.spacing = UX.contentStackViewSpacing
+    }
+
+    private lazy var noAnalysisImageView: UIImageView = .build { imageView in
+        imageView.image = UIImage(named: ImageIdentifiers.shoppingNoAnalysisImage)
+        imageView.contentMode = .scaleAspectFit
+    }
+
+    private lazy var headlineLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .headline,
+                                                            size: UX.headlineLabelFontSize,
+                                                            weight: .bold)
+    }
+
+    private lazy var bodyLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.bodyLabelFontSize)
+    }
+
+    private let footerLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.isUserInteractionEnabled = true
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.footerLabellFontSize)
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupLayout() {
+        let tapGesture = UITapGestureRecognizer(target: self,
+                                                action: #selector(didTapStartAnalysis))
+        footerLabel.addGestureRecognizer(tapGesture)
+        addSubview(cardContainer)
+        addSubview(contentStackView)
+        [noAnalysisImageView,
+         headlineLabel,
+         bodyLabel,
+         footerLabel].forEach(contentStackView.addArrangedSubview)
+
+        NSLayoutConstraint.activate([
+            cardContainer.leadingAnchor.constraint(equalTo: leadingAnchor),
+            cardContainer.topAnchor.constraint(equalTo: topAnchor),
+            cardContainer.trailingAnchor.constraint(equalTo: trailingAnchor),
+            cardContainer.bottomAnchor.constraint(equalTo: bottomAnchor),
+
+            contentStackView.topAnchor.constraint(equalTo: topAnchor,
```

Comment:
The `contentStackView` should not be constraint to the view if it is added to the card view using `configure`. The `configure` method will constraint the view correctly to it's superview. 
However as we need 16 points spacing to the card edges and the card only provides a space of 8 you will probably have to either wrap your stackview in a view and add the missing space or - what I prefer - use a UIView right away and constraint the content to this view with the correct space.